regression/cbmc-incr-smt2/pointers-conversions/pointer_to_int.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
Why is this broken?

Reply:
Additional work is required in the new SMT back-end to make `get_value` work for this will now see arbitrary expressions rather than just symbols. The other back-ends already handle this case properly.

Reply:
Fair.